src/Routes.js

```diff
@@ -25,25 +30,49 @@ export const routes = {
   executorDetails: '/:id/:run_id/:executor_id',
 };
 
-export const Routes = () => (
-  <Suspense fallback={<Fragment />}>
-    <Switch>
-      <Route exact path={routes.home} component={Home} />
-      <Route exact path={routes.details} component={RemediationDetails} />
-      <Route
-        exact
-        path={routes.runDetails}
-        render={(props) => <ActivityDetails remediation={{}} {...props} />}
-      />
-      <Route
-        exact
-        path={routes.executorDetails}
-        render={(props) => <ExecutorDetails {...props} />}
-      />
-      <Redirect path="*" to={routes.home} push />
-    </Switch>
-  </Suspense>
-);
+export const Routes = () => {
+  const [hasSystems, setHasSystems] = useState(true);
+  useEffect(() => {
+    try {
+      axios
+        .get(`${INVENTORY_TOTAL_FETCH_URL}?page=1&per_page=1`)
+        .then(({ data }) => {
+          setHasSystems(data.total > 0);
+        });
+    } catch (e) {
+      console.log(e);
+    }
+  }, [hasSystems]);
```

Comment:
Could the `AppZeroState` component itself not make this request itself?

That way if we’d ever change how we determine when to show the zero state, it would just be in one place to change for all apps.

Reply:
originally the plan was to make the change in the chrome user obj, but this makes sense too. I'll look into this implementation